### ISSUE TITLE
Add support for Paddle and VisualDL

### ISSUE THREAD

**Issue by lamhoangtung** [allegroai/clearml#309]
Recently, there was [Paddle](https://github.com/PaddlePaddle/Paddle), a new deep learning framework that gained a lot of traction in production usage, along with their own equivalent for Tensorboard (or TensorboardX) visualization tools called [VisualDL](https://github.com/PaddlePaddle/VisualDL). We and many company had started to use Paddle and VisualDL in ours DL workflows.

So, it would be great if ClearML can extend the support to Paddle and VisualDL in a future version. For now with our test, everything from Iteration count to metric visualization using VisualDL are not working with ClearML yet.

**Comment by bmartinn** [comment]
Hi @lamhoangtung 
Sure that sounds like a great match for automagic :)
A few things that will help accelerate this process:

1. Simple sample code of Paddle training (maybe something similar to the mnist examples). 
(It should cover model load/save, and logging metrics)

2. GPU / CUDA package matrix link (like the one torch has), in order to have full `clearml-agent` support

BTW:
From the looks of it `VisualDL` is very similar to Tensorboard , so hopefully we could reuse the code :)

**Comment by lamhoangtung** [comment]
Thanks for the interest @bmartinn 

I will try to find some spare time to update a simple sample code of Paddle training soon.

For now, here is the package matrix link, luckily they had something like Pytorch ;)

https://www.paddlepaddle.org.cn/install/quick

**Comment by ainoam** [comment]
Thanks for the example @adri.
We'll look for the right opportunity to add such support.

**Comment by ainoam** [comment]
That's exactly the place @adri.
We'll definitely appreciate your contribution if you're feeling up for it.  If you need any help, head over to the clearml-dev channel on ClearML's [slack](https://joinslack.clear.ml/) space.

**Comment by patrickdasilva-clearml** [comment]
Hi @lamhoangtung , we're closing this issue as stale since it's been created too long ago. If you think the issue is still relevant, please make sure the issue persists on a recent version of ClearML (ideally the latest) before re-opening the issue. We'll gladly look at it again.
